worlds/civ_6/Civ6Client.py

```diff
@@ -0,0 +1,338 @@
+        if isinstance(self.ctx, CivVIContext):
+            print("Toggling progressive eras, stand by...")
+            self.ctx.is_pending_toggle_progressive_eras = True
+
+
+class CivVIContext(CommonContext):
+    is_pending_death_link_reset = False
+    is_pending_toggle_progressive_eras = False
+    command_processor = CivVICommandProcessor
+    game = "Civilization VI"
+    items_handling = 0b111
+    tuner_sync_task = None
+    game_interface: CivVIInterface
+    location_name_to_civ_location = {}
+    location_name_to_id = {}
+    item_id_to_civ_item: Dict[int, CivVIItemData] = {}
+    item_table: Dict[str, CivVIItemData] = {}
+    processing_multiple_items = False
+    received_death_link = False
+    death_link_message = ""
+    death_link_enabled = False
+
+    death_link_just_changed = False
+    # Used to prevent the deathlink from triggering when someone re enables it
+
+    logger = logger
+    progressive_items_by_type = get_progressive_districts_data()
+    item_name_to_id = {
+        item.name: item.code for item in generate_item_table().values()}
+    connection_state = ConnectionState.DISCONNECTED
+
+    def __init__(self, server_address, password, apcivvi_file=None):
+        super().__init__(server_address, password)
+        self.game_interface = CivVIInterface(logger)
+        location_by_era = generate_era_location_table()
+        self.item_table = generate_item_table()
+        self.apcivvi_file = apcivvi_file
+
+        # Get tables formatted in a way that is easier to use here
+        for era, locations in location_by_era.items():
+            for item_name, location in locations.items():
+                self.location_name_to_id[location.name] = location.code
+                self.location_name_to_civ_location[location.name] = location
+
+        for item_name, item in self.item_table.items():
+            self.item_id_to_civ_item[item.code] = item
+
+    async def resync(self):
+        if self.processing_multiple_items:
+            logger.info(
+                "Waiting for items to finish processing, try again later")
+            return
+        await self.game_interface.resync()
+        await handle_receive_items(self, -1)
+        logger.info("Resynced")
+
+    def on_deathlink(self, data: Utils.Dict[str, Utils.Any]) -> None:
+        super().on_deathlink(data)
+        text = data.get("cause", "")
+        if text:
+            message = text
+        else:
+            message = f"Received from {data["source"]}"
+        self.death_link_message = message
+        self.received_death_link = True
+
+    async def server_auth(self, password_requested: bool = False):
+        if password_requested and not self.password:
+            await super(CivVIContext, self).server_auth(password_requested)
+        await self.get_username()
+        await self.send_connect()

```

Comment:
```suggestion
        await self.get_username()
        self.tags = set()
        await self.send_connect()

```
If you add that there, it'll remove the "AP" tag, which is supposed to be only for reference clients

Reply:
Ah nice, thank you 🙏